Server/src/routes/vehicletype.route.ts

```diff
@@ -121,8 +122,13 @@ export class VehicleTypeRoute {
 	}
 
 	private async createType(req: Request, res: Response): Promise<void> {
-		const userData: UpdateVehicleType = req.body
-
+		const userDataPayload = UpdateVehicleType.safeParse(req.body)
+		if (!userDataPayload.success) {
+			logger.error(userDataPayload.error)
+			res.sendStatus(400)
+			return
+		}
+		const userData = userDataPayload.data
 		// TODO: input validation
```

Comment:
This TODO is not necessary anymore, is it?

Reply:
Ay, removed the comment. Thanks!